notifications_utils/clients/redis/bounce_rate.py

```diff
@@ -56,4 +60,4 @@ def get_bounce_rate(self, service_id: str, bounce_window=_twenty_four_hour_windo
             total_notifications_key(service_id), min_score=twenty_four_hours_ago, max_score=now
         )
 
-        return round(total_hard_bounces / (1.0 * total_notifications), 2) if (total_notifications > 0) else 0.0
+        return round(total_hard_bounces / (1.0 * total_notifications), 2) if (total_notifications >= self.minimum_volume) else 0.0
```

Comment:
Won't this hide the problem email percentage from users though?  I thought we were going to separate this off somehow so we could both get the bounce rate but also know whether a particular service's bounce rate is considered "warning/critical".

Also, are going to keep two separate thresholds, one being the display volume threshold, and the other the warning/suspension volume threshold?

Reply:
My misunderstanding, I'll add a new method that will do something similar to `get_bounce_rate` but return the status based on whether the service meets the min volume and leave `get_bounce_rate` as is.